app/scripts/proactive/view/dom.js

```diff
@@ -373,18 +367,9 @@ define(
             	else wfIds += ","
             	wfIds += wfToRemove[wfIndex].get('id');
             }
-            var selectedBucketId = $("#select-bucket").val();
-            window.location.replace('/workflow-catalog/buckets/' + selectedBucketId + '/workflows/' + wfIds + '?alt=zip');
-            
-        })
-        
-        $("#import-archive-button").click(function (event) {
-            event.preventDefault();
-            var studioApp = require('StudioApp');
-            closeCollapsedMenu();
-            $("#import-archive-file").parent('form').trigger('reset');
-            $("#import-archive-file").click();
-        })
+            var selectedBucketId = bucketId ? bucketId : $("#select-bucket").val();
+            return '/workflow-catalog/buckets/' + selectedBucketId + '/workflows/' + wfIds + '?alt=zip';
```

Comment:
Zip files are currently not supported by the workflow-catalog so the url aim to a 415 error Unsupported Media Type. Is it intended ?